lib/p2p/Pool.ts

```diff
@@ -633,6 +637,18 @@ class Pool extends EventEmitter {
       await this.handlePacket(peer, packet);
     });
 
+    peer.on('handshake', (oldHandshakeState) => {
+      // drop all orders for trading pairs that are no longer supported
+      if (oldHandshakeState) {
+        oldHandshakeState.pairs.forEach((pairId) => {
+          if (!peer.pairs!.includes(pairId)) {
+            // a trading pair was in the old handshake state but not in the updated one
+            this.emit('peer.dropPair', peer.nodePubKey!, pairId);
```

Comment:
`dropPair` sounds like a command, not an event. perhaps `pairDropped` / `pairRemoved` / `pairRemoval`.

Reply:
I was trying to follow the imperative style of the `close` event for example, but I also prefer `pairDropped` so I'll go with that.

Reply:
I'm not sure it's the same, because `close` doesn't imply that we should close something. So maybe it's the equivalent of `pairDrop`, but you're the english expert here ;)